README.md

```diff
@@ -9,6 +9,7 @@
 - [Visual Studio Code](#visual-studio-code)
 - [Using NFC](#using-nfc)
 - [License](#license)
+- [Code of Conduct](#Code of Conduct)
```

Comment:
Nice catch as well, I would have forgotten about the table of contents!

I think the anchor link needs to be modified a little bit as it doesn't seem to be rendering:
![image](https://user-images.githubusercontent.com/29712657/112874967-7ddf4380-90bb-11eb-9f45-e84c5e25241f.png)

I believe this will be the fix required, but make sure you check after adding it.
```suggestion
- [Code of Conduct](#code-of-conduct)
```